sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
+    })
+    .catch(err => console.error("Error in insertEmployeeHistory", err));
+};
 
-insertFromCsvLine = (UserID, FundID, PortfolioCompanyName, EmployeeName, OriginalPostion, OriginalStartDate, OriginalEndDate, CurrentEmployer, CurrentPostion, LinkedInUrl, Comments) =>{
-    console.log("\n\n Fields are:, ", UserID, FundID, PortfolioCompanyName, EmployeeName, OriginalPostion, OriginalStartDate, OriginalEndDate, CurrentEmployer, CurrentPostion, LinkedInUrl, Comments, "\n");
-    insertPerson(FundID, EmployeeName, LinkedInUrl, Comments).then((newPerson) => {
-        insertCompany(PortfolioCompanyName, FundID).then((originalCompany) => {
-            insertOriginalFundPosition(newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion).then(() => {
-                //NOTE: I'm hoping it doesn't freak out that I included no arguements here.
-                insertEmployeeHistory(UserID, newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion, OriginalStartDate, OriginalEndDate).then(() => {
-                     insertCompany(CurrentEmployer,FundID).then((newCompany) => {
-                        insertEmployeeHistory(UserID, newPerson.IndividualID, newCompany.CompanyID, CurrentPostion, OriginalStartDate, OriginalEndDate)
-                     });
-                });
-            }).catch(err => console.error(">>ERROR ini insertFromCSVLine, in insertOriginal Position."));
-        });
-    }).catch(err => console.error('Error in "insertFromCsvLine", ', err));
-    
-}
+insertFromCsvLine = (
+  UserID,
+  FundID,
+  PortfolioCompanyName,
+  EmployeeName,
+  OriginalPostion,
+  OriginalStartDate,
+  OriginalEndDate,
+  CurrentEmployer,
+  CurrentPostion,
+  LinkedInUrl,
+  Comments
+) => {
+  console.log(
+    "\n\n Fields are:, ",
+    UserID,
+    FundID,
+    PortfolioCompanyName,
+    EmployeeName,
+    OriginalPostion,
+    OriginalStartDate,
+    OriginalEndDate,
+    CurrentEmployer,
+    CurrentPostion,
+    LinkedInUrl,
+    Comments,
+    "\n"
+  );
+  insertPerson(FundID, EmployeeName, LinkedInUrl, Comments)
+    .then(newPerson => {
+      insertCompany(PortfolioCompanyName, FundID).then(originalCompany => {
+        insertOriginalFundPosition(newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion)
+          .then(() => {
+            //NOTE: I'm hoping it doesn't freak out that I included no arguements here.
+            insertEmployeeHistory(
+              UserID,
+              newPerson.IndividualID,
+              originalCompany.CompanyID,
+              OriginalPostion,
+              OriginalStartDate,
+              OriginalEndDate
+            ).then(() => {
+              insertCompany(CurrentEmployer, FundID).then(newCompany => {
+                insertEmployeeHistory(
+                  UserID,
+                  newPerson.IndividualID,
+                  newCompany.CompanyID,
+                  CurrentPostion,
+                  OriginalStartDate,
+                  OriginalEndDate
```

Comment:
**ISSUE**: Currently, both original and new positions are labeled with the same date. These need to be updated once we figure out quarters of the year.